cdf.toml

```diff
@@ -0,0 +1,32 @@
+[cdf]
+project_dir = "cognite_toolkit"
```

Comment:
Should we consider another word than "project_dir"? I know we've used that until now, but project can be related to both a process and a CDF instance, none of which we are talking about here. How about work_dir or similar?

Reply:
I agree, but I think we do not yet have a good alternative.